### PR TITLE
Change codec settings

### PR DESCRIPTION
**Pull request by rcorrie**
Added this method to the JS api:

```javascript
// Not listed codecs are automatically will have zero priority
endpoint.changeCodecSettings({
  "PCMA/8000/1": 0,
  "G722/16000/1": 0,  // Zero means to disable the codec.
  "iLBC/8000/1": 210,
  "speex/8000/1": 0,
  "speex/16000/1": 0,
  "speex/32000/1": 0
})
```

**NOTE:**
On android, if you enter a codec that cannot be found, the app just crashes. I would like to handle the error and pass it up to the JS side, but I still need to figure it out. @datso maybe you can enlighten me, I am new at native dev and appreciate the help.

resolves https://github.com/Carusto/react-native-pjsip/issues/39

### CODE REVIEW

File: src/Endpoint.js [hunk 1]
```diff
@@ -521,6 +521,18 @@ export default class Endpoint extends EventEmitter {
       NativeModules.PjSipModule.changeOrientation(orientation)
     }
 
+    changeCodecSettings(codecSettings) {
+        return new Promise(function(resolve, reject) {
+        NativeModules.PjSipModule.changeCodecSettings(codecSettings, (successful, data) => {
```
Comment (rcorrie) [inline src/Endpoint.js]: I would like errors to be handled here in this promise. Not sure how to do that. @datso

**Comment by rcorrie** [comment]
@datso any plans on merging this one as well?

**Comment by datso** [comment]
Could you please fix the conflict ? It happens after the merge of the another one ?
Thanks btw for the PR.

**Comment by rcorrie** [comment]
@datso done

**Comment by datso** [comment]
thanks, merged.

**Pull request merged**